lib/modules/dosomething/dosomething_reportback/includes/ReportbackItemTransformer.php

```diff
@@ -40,11 +40,21 @@ public function index($parameters) {
   public function show($id) {
     try {
       $reportbackItem = ReportbackItem::get($id);
-      $reportbackItem = $this->removeUnauthorizedResults($reportbackItem);
+
+      if (!user_access('administer modules')) {
+        $reportbackItem = $this->removeUnauthorizedResults($reportbackItem, TRUE);
+      }
```

Comment:
@weerd I think that is a good idea! Removed the params logic and tweaked `removeUnauthorizedResults` and the `show` function in `ReportbackTransformer.php` to more closely represent what is going on in `dosomething_reportback_accessible_results()` - let me know  what you think!